### DISCUSSION TITLE
230529~230604 회고

### DISCUSSION THREAD

**Discussion by chairoi** [BDD-CLUB/challenge-every-week-posting#47] [5. 회고]
일주일간 무엇을 했는지 간단히 회고하는 글입니다.

[230529~230604 회고](https://thisischacha.tistory.com/4)

**Comment by pnudpwls1103** [comment]
회고를 보면 충분히 열심히 살았는데 슬퍼하지 않았으면 좋겠네요 ㅎㅎ
2학기에는 좀 더 중요한 일에 집중하면서 원하는 결과 차근차근 얻어가시면 좋겠습니다 :)
